### PR TITLE
Properly set sources for arm32

### PR DESCRIPTION
**Pull request by nox**
SKIA_OPTS_ARM_SRC doesn't exist.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/skia/111)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -727,7 +727,7 @@ if($ENV{TARGET} MATCHES "(i386|x86_64)-.*")
     endif()
   endif()
 elseif($ENV{TARGET} MATCHES "arm-.*")
-  set(SKIA_SRC ${SKIA_SRC} ${SKIA_OPTS_ARM_SRC})
+  set(SKIA_SRC ${SKIA_SRC} ${SKIA_OPTS_ARM_NEON_SRC})
```
Comment (emilio) [inline CMakeLists.txt]: Isn't this including all the neon sources unconditionally?

Comment (nox) [inline CMakeLists.txt]: Mmmh, maybe.

I guess we should steal the magic from the gyp files, but then I wonder why we don't just use gyp.

Comment (nox) [inline CMakeLists.txt]: @emilio No it seems ok according to https://github.com/servo/skia/blob/arm32/gyp/opts.gyp#L88-L92.

Whether neon support is optional or not, we add the files pulled in by opts_neon listed there: https://github.com/servo/skia/blob/arm32/gyp/opts.gyp#L233-L242.

And it seems we do ask for optional neon support: 
```
  add_definitions(-mfpu=neon -D__ARM_HAVE_OPTIONAL_NEON_SUPPORT)
```

Comment (metajack) [inline CMakeLists.txt]: We don't use gyp to avoid adding a third build system to our prerequisites.

**Pull request merged**